pkgs/development/tools/hover/default.nix

```diff
@@ -0,0 +1,70 @@
+{ buildGoModule, fetchFromGitHub, fetchurl, glfw, libGL, libX11, libXcursor, libXi,
+  libXinerama, libXrandr, libXxf86vm, makeWrapper, pkgconfig, stdenv, unzip}:
+
+buildGoModule rec {
+  pname = "hover";
+  version = "unstable-2020-01-23";
+
+  subPackages = [ "." ];
+
+  modSha256 = "019lg3s86965jyqfy83hb52y5rb6g91prsg3ckjxbcnca68lsdja";
+
+  src = fetchFromGitHub {
+    rev = "a6f11fe3c4946c9e7cce75360902bd79a077ccf4";
+    owner = "go-flutter-desktop";
+    repo = pname;
+    sha256 = "0pwdzd34zcbyspl7qiwfcrppr6zxp2zliqg9n8s6s3l6k7mffl9j";
+  };
+
+  libFlutter = fetchurl {
+    name = "libflutter_engine.zip";
+    url = "https://storage.googleapis.com/flutter_infra/flutter/051762798888ebea4666c5f9c9f4c0e4d3b3b481/linux-x64/linux-x64-embedder";
```

Comment:
This won't work on Darwin, or on arm. You should either pull this for each arch or limit the platforms of this project.

Reply:
I changed to get binaries different for darwin. Thanks!